### PR TITLE
Add `vansh.dino.icu`

### PR DESCRIPTION
**Pull request by lowkeyvansh**
Using dino.icu subdomain for personal website!

### CODE REVIEW

**Review by reesericci — CHANGES_REQUESTED** [review]
CNAME records like this don't support URL paths, they aren't like website redirects.

You should go into the Pages settings of the github repo, and connect your dino.icu domain as a custom domain. Here's docs that explain how to do that better:

https://docs.github.com/en/pages/configuring-a-custom-domain-for-your-github-pages-site/managing-a-custom-domain-for-your-github-pages-site

**Comment by lowkeyvansh** [comment]
Done!

**Comment by reesericci** [comment]
The commit hasn't been updated to reflect that, it should match what it says in the github pages settings under the custom domain

**Comment by lowkeyvansh** [comment]
guLiO31

i have updated it in github pages settings! any suggestions on how to fix this?

File: dino.icu.yaml [hunk 1]
```diff
@@ -331,6 +331,12 @@ tejas: # by tej
   type: CNAME
   value: tejasag.netlify.app.
 
+vansh:
+ - ttl: 1
+   type: CNAME
+   value: lowkeyvansh.github.io/web.
```
Comment (Muirrum) [inline dino.icu.yaml:337]: You need to remove the "/web" from this line

Comment (lowkeyvansh) [reply dino.icu.yaml]: updated^

**Review by reesericci — APPROVED** [review]
Looks good to me!

**Pull request merged**